dts/riscv/telink/telink_b91.dtsi

```diff
@@ -40,6 +40,15 @@
 			compatible = "mmio-sram";
 		};
 
+		mtimer: timer@e6000000 {
+			compatible = "riscv,machine-timer";
+			reg = <0xe6000000 0x8
+			       0xe6000008 0x8>;
+			interrupts = <7 0>;
```

Comment:
what the interrupt `0` is for? That should be the `User software interrupt` but is that used anywhere?

Reply:
`0` is the priority, unless this platform does it differently

Reply:
hahaha, of course. I'll go hiding in shame, see ya.